src/urllib3/connection.py

```diff
@@ -74,7 +74,7 @@ class BaseSSLError(BaseException):  # type: ignore
 _CONTAINS_CONTROL_CHAR_RE = re.compile(r"[^-!#$%&'*+.^_`|~0-9a-zA-Z]")
 
 
-HTTPBody = Union[bytes, IO[Any], Iterable[bytes], str]
+_TYPE_HTTP_BODY = Union[bytes, IO[Any], Iterable[bytes], str]
```

Comment:
For consistency let's name this `_TYPE_BODY`

Reply:
Done.